src/java.base/share/classes/jdk/internal/javac/PreviewFeature.java

```diff
@@ -64,11 +64,6 @@
      * Values should be annotated with the feature's {@code JEP}.
      */
     public enum Feature {
-        // not used, but required for interim javac to not warn.
```

Comment:
Maybe we should update this comment to note their removal depends on the boot JDK version?